benchmark_utils/backbones_architecture.py

```diff
@@ -33,6 +34,25 @@ def forward(self, x, sample_weight=None):
 
         return x
 
+class OfficeConvNet(nn.Module):
```

Comment:
this neural network is not specific to the office dataset, right? can you call it `ResNet50` ?